Parse/src/main/java/com/parse/Parse.java

```diff
@@ -620,7 +627,12 @@ private static void initializeParseHttpClientsWithParseNetworkInterceptors() {
     interceptors.add(interceptor);
   }
 
-  // TODO(mengyan) Make public after we release the interceptor feature
+  /**
+   * Remove a given {@code ParseNetworkInterceptor}. You must invoke
+   * {@code Parse.removeParseNetworkInterceptor} before {@code Parse.initialize}.
```

Comment:
ideally this should be `{@link Parse#removeParseNetworkInterceptor()}` though I'm not super familiar with the syntax.

Also it should be `{@link Parse#initialize()}.`